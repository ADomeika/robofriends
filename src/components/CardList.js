import React from 'react';

import Card from './Card';

const CardList = ({ robots }) => {
  return (
    <div>
      {robots.map((robot) => (
        <Card key={robot.id} item={robot} />
      ))}
    </div>
  );
};
export default CardList;
