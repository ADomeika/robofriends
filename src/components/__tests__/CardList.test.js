import React from 'react';
import { shallow } from 'enzyme';
import CardList from '../CardList';

const mockRobots = [
  {
    id: 1,
    name: 'Test Tester',
    username: 'TeTester',
    email: 'test@test.com',
  },
];

describe('<CardList />', () => {
  it('should render', () => {
    expect(shallow(<CardList robots={mockRobots} />)).toMatchSnapshot();
  });
});
