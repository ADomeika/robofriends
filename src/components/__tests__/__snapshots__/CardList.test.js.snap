// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CardList /> should render 1`] = `
<div>
  <Card
    item={
      Object {
        "email": "test@test.com",
        "id": 1,
        "name": "Test Tester",
        "username": "TeTester",
      }
    }
    key="1"
  />
</div>
`;
